media-gfx/asymptote/asymptote-2.61.ebuild

```diff
@@ -8,7 +8,7 @@ PYTHON_COMPAT=( python3_{6,7} )
 inherit autotools elisp-common latex-package python-r1
 
 DESCRIPTION="A vector graphics language that provides a framework for technical drawing"
-HOMEPAGE="http://asymptote.sourceforge.net/"
+HOMEPAGE="https://asymptote.sourceforge.net/"
```

Comment:
Does it work for you? I currently can only reach ```http```, but ```https``` give timeout.

Reply:
Since I can't reach it also from other PCs, I will drop this commit and merge the rest.

Reply:
Sorry for the late reply, you're right, the change was wrong. I forgot to change the ending to `.io` too.. i'll create a new PR.